tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +111,22 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
     # Assertions should check:
     # - That result is truthy
+    assert result
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) is 3
+    assert len(jesse.inventory) is 3
```

Comment:
We should use `==` instead of `is` when checking for value equality. Here's a helpful article that describes the differences:

https://towardsdatascience.com/whats-the-difference-between-is-and-in-python-dc26406c85ad